ca_gazebo/package.xml

```diff
@@ -13,7 +13,11 @@
   <build_depend>gazebo_plugins</build_depend>
   <build_depend>roscpp</build_depend>
   <build_depend>ca_msgs</build_depend>
+  <build_depend>apriltag_ros</build_depend>
+  <build_depend>tf</build_depend>
 
+  <run_depend>apriltag_ros</run_depend>
```

Comment:
If a dependency is both a run_depend and build_depend, the it is convenient to just use `<depend>`. See [here](http://wiki.ros.org/catkin/package.xml#Dependencies)